server/server.js

```diff
@@ -370,6 +380,24 @@ app.get(`${apiBaseUrl}/employer-sites`,
     return res.json({ data: result });
   }));
 
+app.get(`${apiBaseUrl}/employer-sites/:id`,
+  keycloak.allowRolesMiddleware('health_authority', 'ministry_of_health'),
+  keycloak.getUserInfoMiddleware(),
+  asyncMiddleware(async (req, res) => {
+    const user = req.hcapUserInfo;
+    const [result] = await getSiteByID(req.params.id);
+    logger.info({
+      action: 'employer-sites-detail_get',
+      performed_by: {
+        username: user.username,
+        id: user.id,
+      },
+      eeoi_id: req.query.id,
```

Comment:
Not sure what this field is tracking - I think we're only accessing a site from this endpoint and not an EEOI right?